pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerAdminApiApplication.java

```diff
@@ -70,7 +98,12 @@ private void setupSwagger() {
     beanConfig.setDescription("APIs for accessing Pinot broker information");
     beanConfig.setContact("https://github.com/linkedin/pinot");
     beanConfig.setVersion("1.0");
-    beanConfig.setSchemes(new String[]{"http"});
+    if (_useHTTPS) {
+      beanConfig.setSchemes(new String[]{"https"});
+    }
```

Comment:
please move up the else to this line, as per coding conventions.  See https://pinot.readthedocs.io/en/latest/dev_guide.html on how to set up your dev guide so that coding conventions are automatic. thanks.